package.json

```diff
@@ -23,6 +23,7 @@
     "aframe-billboard-component": "^1.0.0",
     "aframe-extras": "https://github.com/MozillaReality/aframe-extras#feature/precompute-nav-mesh",
     "aframe-input-mapping-component": "https://github.com/mozillareality/aframe-input-mapping-component#hubs/master",
+    "aframe-motion-capture-components": "https://github.com/mozillareality/aframe-motion-capture-components#1ca616fa67b627e447b23b35a09da175d8387668",
```

Comment:
Points to a branch with support for A-Frame 0.8.0. See https://github.com/dmarcos/aframe-motion-capture-components/pull/40